.eslintrc.json

```diff
@@ -0,0 +1,16 @@
+{
```

Comment:
### Personal and subjective comment alert!

I prefer the eslintrc to be a javascript file.

 - Is smaller
 - Allows comments
 - No json shitty errors because commas!

Reply:
We had the same issue between .eslintrc and .eslintrc.json :laughing:

@CartoDB/frontend any comment?